tests/automated_unit_tests/comms/test_user_settings_manager.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+
+from asmcnc.comms.logging_system.logging_system import Logger
+from asmcnc.comms.user_settings_manager import UserSettingsManager
+
+sys.path.append('./src')
+
+try:
+    import unittest
+    import pytest
+    from mock import Mock, MagicMock
+
+except:
+    Logger.info("Can't import mocking packages, are you on a dev machine?")
+
+'''
+######################################
+RUN FROM easycut-smartbench FOLDER WITH: 
+python -m pytest tests/automated_unit_tests/comms/test_user_settings_manager.py
+######################################
+'''
+
+
+class UserSettingsManagerTests(unittest.TestCase):
+    def test_code_coverage(self):
+        #  make sure the file does not exist:
+        try:
+            os.remove(UserSettingsManager.SETTINGS_FILE_PATH)
+        except:
+            # file seems to be gone already...
+            pass
+        # file should not exist anymore:
+        assert(not os.path.exists(UserSettingsManager.SETTINGS_FILE_PATH))
+        usm = UserSettingsManager()  # make new instance with default values
+        usm.save_settings_to_file()
+        # file should exist again:
+        assert(os.path.exists(usm.SETTINGS_FILE_PATH))
+
+        # getters:
+        value = usm.get_value('dust_shoe_detection')
+        assert value
+        title = usm.get_title('dust_shoe_detection')
+        assert (title == 'Dust shoe plug detection')
```

Comment:
redundant paranthesis in a couple places

Reply:
👍  (Trusted pyCharm to tell me about redundant paranthesis...)